.circleci/config.yml

```diff
@@ -24,18 +23,26 @@ jobs:
           name: Check formatting
           command: cargo make check-rs-format
       - restore_cache:
-          key: dependency-rs-sccache
+          keys:
+            # Find a cache corresponding to this specific Cargo.lock checksum
+            # Find the most recently generated cache
+            - dependency-rs-sccache-{{ checksum "Cargo.lock" }}
+            - dependency-rs-sccache-
       - restore_cache:
-          key: dependency-rs-cache-{{ checksum "Cargo.lock" }}
+          keys:
+            - dependency-rs-cache-{{ checksum "Cargo.lock" }}
+            - dependency-rs-cache-
       - restore_cache:
-          key: dependency-js-cache-{{ checksum "./api_tests/package.json" }}
+          keys:
+            - dependency-js-cache-{{ checksum "./api_tests/package.json" }}
+            - dependency-js-cache-
       - run:
           name: Build using cargo make
           command: |
             cargo --version
-            cargo make travis
+            cargo make ci
       - save_cache:
-          key: dependency-rs-sccache
+          key: dependency-rs-sccache-{{ checksum "Cargo.lock" }}
```

Comment:
I think we should evaluate, whether or not we actually still want `sccache`. Since caches are separate and dependent on the `Cargo.lock` and not our source-code, we should be able to skip uploading the cache unless we change dependencies.